config/routes.rb

```diff
@@ -7,7 +7,7 @@
 
   resources :posts, only: [:new, :create]
   resource :user_session, only: [:new, :create, :destroy]
-  resources :users, only: [:new, :create] do
+  resources :users, only: [:new, :create, :show, :edit, :update] do
```

Comment:
I also think that this should be changed so that `edit` and `update` are on a `resource :user`.  That way there is no possibility of  URL hacking to edit a particular user.